app/src/test/groovy/pl/futurecollars/innvoicing/db/memory/FileDatabaseTest.groovy

```diff
@@ -0,0 +1,6 @@
+package pl.futurecollars.innvoicing.db.memory
+
+import pl.futurecollars.innvoicing.db.memory.InMemoryDatabase
+
+class FileDatabaseTest extends InMemoryDatabase {
```

Comment:
to powinno rozszerzać AbstractDatabaseTest a nie InMemoryDatabase :)

Reply:
Tak, ale ja nie mam pliku FileDatabase, w konsekwencji nie działające testy bo nie ma co testować